### ISSUE TITLE
Debug Failure. Did not expect ClassDeclaration to have an Identifier in its trivia

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#35133]
**TypeScript Version:**  3.8.0-dev.20191115

**Repo:**
1. Open https://github.com/microsoft/vscode/files/3852933/extension-prod.zip in VS Code
1. Request document outline

**Bug:**

```
<syntax> TypeScript Server Error (3.8.0-dev.20191115)
Debug Failure. Did not expect ClassDeclaration to have an Identifier in its trivia
Error: Debug Failure. Did not expect ClassDeclaration to have an Identifier in its trivia
    at addSyntheticNodes (tsserver.js:129820:30)
    at createChildren (tsserver.js:129809:9)
    at NodeObject.getChildren (tsserver.js:129753:56)
    at Object.findChildOfKind (tsserver.js:102980:26)
    at spanForNode (tsserver.js:113685:36)
    at getOutliningSpanForNode (tsserver.js:113634:28)
    at visitNonImportNode (tsserver.js:113491:28)
    at visitNodes (tsserver.js:17638:30)
    at Object.forEachChild (tsserver.js:17869:24)
    at NodeObject.forEachChild (tsserver.js:129776:23)
    at visitNonImportNode (tsserver.js:113504:23)
    at visitNode (tsserver.js:17629:24)
    at Object.forEachChild (tsserver.js:17759:21)
    at NodeObject.forEachChild (tsserver.js:129776:23)
    at visitNonImportNode (tsserver.js:113504:23)
    at visitNode (tsserver.js:17629:24)
    at Object.forEachChild (tsserver.js:17832:24)
    at NodeObject.forEachChild (tsserver.js:129776:23)
    at visitNonImportNode (tsserver.js:113504:23)
    at visitNode (tsserver.js:17629:24)
    at Object.forEachChild (tsserver.js:17849:24)
    at NodeObject.forEachChild (tsserver.js:129776:23)
    at visitNonImportNode (tsserver.js:113504:23)
    at visitNode (tsserver.js:17629:24)
    at Object.forEachChild (tsserver.js:17880:24)
    at NodeObject.forEachChild (tsserver.js:129776:23)
    at visitNonImportNode (tsserver.js:113504:23)
    at addNodeOutliningSpans (tsserver.js:113466:21)
    at Object.collectElements (tsserver.js:113454:13)
    at Object.getOutliningSpans (tsserver.js:131097:50)
    at IOSession.Session.getOutliningSpans (tsserver.js:140318:45)
    at Session.handlers.ts.createMapFromTemplate._a.<computed> (tsserver.js:139414:61)
    at tsserver.js:140991:88
    at IOSession.Session.executeWithRequestId (tsserver.js:140982:28)
    at IOSession.Session.executeCommand (tsserver.js:140991:33)
    at IOSession.Session.onMessage (tsserver.js:141014:35)
    at Interface.<anonymous> (tsserver.js:142329:27)
    at Interface.emit (events.js:200:13)
    at Interface._onLine (readline.js:314:10)
    at Interface._normalWrite (readline.js:459:12)
    at Socket.ondata (readline.js:170:10)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:166:17): Error: <syntax> TypeScript Server Error (3.8.0-dev.20191115)
```

**Comment by 5saviahv** [comment]
This error seems to flood my "log (window)" in VScode. I tried to understand what may cause this error. I realised It seems to happen if I declare anything through prototype.

It happens even with this small class: 
``` javascript
class Vaata {
    constructor(name = "") {
        this.name = name;
    }
}

/* Set simple signature for the object */
Vaata.prototype.vaata = '[object Vaata]';
```
This error affects vscode some unexpected way. Somehow not when loaded but some time later, all code folding artifacts from UI go missing.
